code/code/misc/weather.cc

```diff
@@ -751,7 +746,7 @@ void Weather::weatherChange()
   // keep in range 1040 - 960
 
   int diff = 0;
-  changeWeatherT change;
+  Weather::changeWeatherT change = Weather::CHANGE_NONE;
```

Comment:
Could you explain why `Weather::` is necessary in `Weather::changeWeatherT`?

Reply:
Nope haha, I can't explain it as I don't know the answer (though I'm guessing the answer is "it's not necessary"?). That was just how they declared it when they initialized the same variable elsewhere so I wrote it the same way here. Would they have written it that way elsewhere because they weren't inside the Weather namespace in that code?

Reply:
Class, not namespace - otherwise yes. You don't need that prefix inside the class. Though you added it, so I was assuming that I'm missing something, and thought perhaps there's some reason.